skimage/segmentation/slic_superpixels.py

```diff
@@ -11,7 +11,7 @@
 def slic(image, n_segments=100, compactness=10., max_iter=10, sigma=0,
          spacing=None, multichannel=True, convert2lab=None,
          enforce_connectivity=True, min_size_factor=0.5, max_size_factor=3,
-         slic_zero=False):
+         slic_zero=False, n_jobs=1):
```

Comment:
I think we should mention this in https://github.com/scikit-image/scikit-image/blob/master/doc/release/release_dev.rst .